scripts/run-system-tests

```diff
@@ -54,12 +54,12 @@ while getopts 's:hknr:?' flag; do
   s) SEARCH=${OPTARG:-'*'} ;;
   h) CYPRESS_HEADED="true" ;;
   n) CYPRESS_NO_EXIT="true" ;;
-  k) NO_BUILD="true" ;;
+  d) DEVELOPMENT="true" ;;
   r) RETRIES=${OPTARG} ;;
   ?)
-    echo -e "+-------------------------------+"
-    echo -e "| DC/OS Integration test runner |"
-    echo -e "+-------------------------------+\n"
+    echo -e "+--------------------------+"
+    echo -e "| DC/OS System test runner |"
+    echo -e "+--------------------------+\n"
     echo -e "Don't forget running npm run build:assets first!\n\n"
     echo -e "Usage: \n"
     echo -e "\tnpm run test:system -- [-s <FILE_NAME>] [-hnk] [-r <RETRIES>]\n"
```

Comment:
still outdated `-k` and missing `-d` (same for integration tests)

Reply:
Will do 👍